src/style.css

```diff
@@ -2,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, Arial, Helvetica, sans-serif, Verdana;
+  box-sizing: border-box;
+  outline: none;
+  display: block;
+  width: 100%;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search__input::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.search__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search__form[data-qa='big'] .search__input {
+  font-size: 16px;
+  background-size: 19px;
+  background-position: 25px center;
+  padding-left: 62px;
+  height: 70px;
```

Comment:
As per the checklist, fixing the height of a container is not recommended unless there is a specific requirement for it. It's generally better to let the content size dictate the container size to avoid overflow or accidental scroll bars. However, if you have a specific reason for setting the height in this case, that's fine.